examples/team-enable.py

```diff
@@ -0,0 +1,136 @@
+import getpass
+from gremlinapi.config import GremlinAPIConfig as config
+from gremlinapi.clients import GremlinAPIClients as clients
+from gremlinapi.orgs import GremlinAPIOrgs as orgs
+import sys
+
+def enable_all(team_id, targets) -> bool:
+    success = True
+    for rover in targets:
+        if clients.activate_client(teamId=team_id, guid=rover['identifier']) != 'Success':
+            success = False
+    return success
+
+def disable_all(team_id, targets) -> bool:
+    success = True
+    for rover in targets:
+        if clients.deactivate_client(teamId=team_id, guid=rover['identifier']) != 'Success':
+            success = False
+    return success
+
+def confirm_yes() -> bool:
+    while True:
+        answer = input("Proceed?  Enter \"yes\" or \"no\": ").casefold()
+        if answer == 'yes':
+            return True
+        elif answer == 'no':
+            return False
+        else:
+            print(f"""\"{answer}\" is not valid.""")
+
+def interactive_enable_agents() -> bool:
+    print("""
+This application needs your Gremlin API Key.  Managing Gremlin API Keys is done at
+https://app.gremlin.com/profile/apikeys
+
+The next prompt, where you enter your Gremlin API Key, does not echo what you type to help keep your
+Gremlin API Key protected.  Ctrl + Shift + V works to paste in Linux and Windows.  Command + V works
+to paste in macOS.  Your Gremlin API Key can be copied from the webpage by clicking the double
+document copy button next to the API Key name then clicking the popup.
+""")
+    key = getpass.getpass('Enter your Gremlin API Key: ')
+    config.api_key = 'Key 8b4500db2d49008c5a89336b0cef41ca0e08ae5b0f8034497fd2cc61b1f20a5e'

```

Comment:
Can you revoke this key and replace it with a dummy value

Reply:
DOH!  That was stupid.  And a bug.

Done.  And thanks!